algebra/src/to_field_vec.rs

```diff
@@ -77,12 +78,13 @@ impl<M: SWModelParameters, ConstraintF: Field> ToConstraintField<ConstraintF> fo
             x_fe.extend_from_slice(&y_fe);
             Ok(x_fe)
         } else {
-            // Otherwise, return the coordinates of the infinity representation in Jacobian
-            let mut x_fe = self.x.to_field_elements()?;
-            let y_fe = self.y.to_field_elements()?;
+            // Otherwise, serialize the point as the affine point x=0, y=0,
+            // which is for sure not on the curve unless b=0 (which should never happen in our case
+            // as if b=0 then the curve has non-prime order).
+            debug_assert!(!M::COEFF_B.is_zero());
```

Comment:
Ok, as agreed offline, I will add a ToDo specifying that in case we will ever need to employ a curve with b=0, we can employ (0,1) as a representation for the point at infinity